params/protocol_params.go

```diff
@@ -119,6 +119,7 @@ const (
 	ElasticityMultiplier     = 2          // Bounds the maximum gas limit a block may have.
 	InitialBaseFee           = 1 * GWei   // Initial base fee for blocks.
 	MaxBaseFee               = 100 * GWei // Maximum base fee for blocks.
+	InitialStateLimit        = 1 * GWei   // Initial state fee for blocks.
```

Comment:
this is a state price, not a state limit

Reply:
oops, changing it to a real value, its only used for genesis block

Reply:
Changed this value to actual gas value, currently i have set the ceil to 40M and Initial limit to 5M